### PR TITLE
homepage redesign

### PR DESCRIPTION
**Pull request by cathydeng**
## Changes
updating homepage design to be simpler and more mobile-friendly

@admoorgit can you confirm that the spacing around things looks as you'd expect on various screen sizes?

## Screenshots
localhost_3000_(Nest Hub Max)

localhost_3000_(iPhone XR) (1)

## Checklist

- [ ] if adds a new page, adds accessibility tests
- [ ] if adds a new page or new interaction, adds e2e test
- [ ] if adds a new page or new interaction, sends google analytics events and updates [GA doc](https://docs.google.com/document/d/1DfQDUNZPO3B352EhUwXp0el6qrAWz8Jq9cTa1yv8Ty4/edit)
- [ ] if changes filter functionality, updates [filters doc](https://docs.google.com/document/d/1yEdo7IpHCtEKNNF6FMLapBUgcPw_8CY7wGwxKTdtJrU/edit)

Fixes https://app.asana.com/0/1202105544020589/1203031893746137/f

### CODE REVIEW

**Review by admoorgit — CHANGES_REQUESTED** [review]
Wow! This is looking really nice @cathydeng. Spacing overall looks solid at different widths already. Here are a few nits:
- Let's shrink the hands icon to 56w
- In doing so, we can shave down the space between it and the guided experience content to 2rem 
image
- Looks like I need to get you a wider strip of faces. It cuts off awkwardly at larger widths 
image
- Wonder if it'll be less awk if it's centered?

**Comment by cathydeng** [comment]
@admoorgit i updated the bg image for desktop. the original image looked pretty awk on smaller desktop windows b/c it took up a lot of space (tricky to make it not overlap w text and also cover full height of section) so i ended up just shaving off one column of faces to the right, which seems to look reasonable at diff screen sizes

File: src/components/Home/ContentCard.tsx [hunk 1]
```diff
@@ -1,35 +1,29 @@
...
-      <div className="usa-card__header-alt height-15 display-flex flex-justify-center flex-align-center">
-        {headerContent}
+    <Grid tablet={{ col: true }}>
+      <div className="margin-y-3">
+        <div className="height-8 display-flex flex-justify-center flex-align-center">
+          {logo}
+        </div>
+        <h2 className="margin-y-2">{header}</h2>
+        <p>{body}</p>
+        {
```
Comment (jlhogan) [inline src/components/Home/ContentCard.tsx:20]: why `{` here?

File: src/pages/Home.tsx [hunk 2]
```diff
@@ -1,166 +1,121 @@
...
-            </Grid>
             <Grid col={12}>
-              <div className="display-none tablet:display-block" aria-hidden>
-                <div className="radius-lg bg-white padding-4 display-flex desktop:padding-y-4 tablet:margin-x-2 desktop:margin-x-0">
-                  <ZipCard id="desktop_zip" />
-                  <VerticalLineText>{t("or")}</VerticalLineText>
-                  <GuidedSearchCard isMobile={false} />
-                </div>
+              <div className="padding-y-2">
+                {showCrisisAlert && (
```
Comment (jlhogan) [inline src/pages/Home.tsx:60]: previously the spec was "show this unless 1) user has closed it 2) user is in guided search flow" just confirming 2 no longer desired

**Review by jlhogan — APPROVED** [review]
noice

File: src/pages/Home.tsx (hunk 2 above)
Comment (cathydeng) [reply src/pages/Home.tsx:60]: yep - discussed w adam and we're ok w keeping this on just the homepage

File: src/components/Home/ContentCard.tsx (hunk 1 above)
Comment (cathydeng) [reply src/components/Home/ContentCard.tsx:20]: ah we used to pass in the entire link component, but now that i update the props this isn't necessary, thx!

**Review by karynlu — APPROVED** [review]

**Review by admoorgit — CHANGES_REQUESTED** [review]
Hello @cathydeng!
Here's a new "wide" hero image with the last column shaved off. This'll make it a little less tight with the content before that smaller width kicks in. 

Here are 2 higher res versions of the 'large' and 'small' hero images.
hero-small
hero-lg

**Review by admoorgit — CHANGES_REQUESTED** [review]
This works for me!

**Review by admoorgit — APPROVED** [review]
This works for me! (disregard previous — errant click! APPROVED!)

**Pull request merged**